client/src/containers/Organisation/Organisation.js

```diff
@@ -16,26 +19,40 @@ class Organisation extends Component<Props> {
     this.state = { organisation: {} };
   }
   componentDidMount() {
-    this.getOrganisationDetail();
+    this.getCurrentOrganisation();
   }
 
-  getOrganisationDetail = () => {
+  getCurrentOrganisation = () => {
     const organisationId = this.props.match.params.organisation_id;
     const { dispatch } = this.props;
-    dispatch(fetchOrganisationDetail(organisationId));
+    dispatch(fetchCurrentOrganisation(organisationId));
   };
 
   render() {
-    const { organisation } = this.props.organisationDetail;
+    const { organisation } = this.props.currentOrganisation;
     return (
       <div>
-        {organisation && !organisation[0].detail ? (
+        {!isEmpty(organisation) ? (
           <Container>
-            <CardImg top width="100%" src={organisation[0].logo} />
-            <ContentHeader heading={organisation[0].name} />
+            <MetaTags>
```

Comment:
@taleymirza @JavaEria this can be a component on its own.

Reply:
@mashhoodr should we create another branch to make it a separate component?

Reply:
yes